packages/kbn-optimizer/src/common/dll_manifest.ts

```diff
@@ -19,7 +19,7 @@ export interface ParsedDllManifest {
 }
 
 const hash = (s: string) => {
-  return Crypto.createHash('md5').update(s).digest('base64').replace(/=+$/, '');
+  return Crypto.createHash('sha1').update(s).digest('base64').replace(/=+$/, '');
```

Comment:
Mind if we split the sha1 changes to a different PR?

Reply:
Opened #190642, thanks.